src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -747,4 +747,32 @@ Parameter name: {0}</value>
   <data name="DuplicatedNewerPackage" xml:space="preserve">
     <value>A newer version of the package called {0} version {2} was found at {1} with version {3}. The newer version has been ignored.</value>
   </data>
+  <data name="PackageStatePendingUninstall" xml:space="preserve">
+    <value>Scheduled for uninstall</value>
+  </data>
+    <data name="PackageStateError" xml:space="preserve">
+    <value>Error</value>
+  </data>
+    <data name="PackageStateLoaded" xml:space="preserve">
+    <value>Loaded</value>
+  </data>
+  <data name="PackageStateUnloaded" xml:space="preserve">
+    <value>Skipped</value>
+  </data>
+  <data name="PackageStatePendingUninstallTooltip" xml:space="preserve">
+    <value>Scheduled for uninstall.
+This package will be uninstalled after the next Dynamo restart.</value>
+  </data>
+    <data name="PackageStateErrorTooltip" xml:space="preserve">
+    <value>Error.
+This package has not been installed due to an unexpected error.</value>
+  </data>
+    <data name="PackageStateLoadedTooltip" xml:space="preserve">
+    <value>Loaded.
+This package is loaded and ready to be used.</value>
+  </data>
+    <data name="PackageStateUnloadedTooltip" xml:space="preserve">
+    <value>Skipped.
+This package has not been loaded because another conflicting package was loaded before it.</value>
+  </data>
```

Comment:
@mjkkirschner @Amoursol @Jingyi-Wen
It has been brought to my attention that the terms used are not really consistent (like loaded vs installed)
What about using the following terms:
1. **Installed** - package is loaded into Dynamo and ready to be used.
2. **Scheduled for uninstall** - package will be uninstalled after next Dynamo restart.
3. **Skipped** - package is available on disk but was not loaded into Dynamo due to conflict with another similar package.
4. **Error** - package was not loaded into Dynamo due to an unexpected error

Also what do you think about scheduled vs marked ....for uninstall ?

If you have any alternatives....please tag them with the numbers above so that we know which one is which :)

Reply:
1. I would use **Loaded** vs. Installed since it implies that the package needs to be downloaded as well as loaded in one word.
2. I wish to clarify what it means to say **Scheduled for uninstall** - does it mean scheduled for unload or scheduled to be nuked? If not clear, we should probably change it to say **Scheduled for deletion** or **Scheduled for unloading**
3. Instead of skipped why don't we just use the term **Unloaded**?

Reply:
The main concern, I guess, is about using **Loaded**/**Unloaded** vs Scheduled for **uninstall**.
What @aparajit-pratap is suggesting is consistent and I would say they seem fine to me

Reply:
1. I agree with @aparajit-pratap on using **Loaded**, since we call the window "Installed packages". And generally I think **install and uninstall** sounds more like user behaviors while **load and unload** sounds more like Dynamo behaviors
2.  Agree on **Scheduled for unloaded** or **deletion**. Same reason above
3.  **Not loaded** or **Unloaded** makes sense to me

Reply:
1. I'm also in favour of **Loaded** as it uses stronger language (Installed could still theoretically fail to load)
2. _Scheduled for Uninstall/Unload/Deletion_ is still the one that feels most ambiguous, but I don't have a better suggestion. In lieu of that I would lean towards **Scheduled for Deletion/Unload** as above.
3. **Unloaded** makes most sense to me here to match the inverse of the _Loaded_ state.
4. **Error** is perfect here - can the user see the error message and do something about it in this state @pinzart?

Reply:
@Amoursol Regarding the error state. Hopefully we can display something meaningful to the user....but that work will probably be a part of this jira task https://jira.autodesk.com/browse/DYN-3670